libraries/AP_Mount/AP_Mount_MAVLink.cpp

```diff
@@ -25,23 +25,7 @@ void AP_Mount_MAVLink::update()
 
     update_mnt_target();
 
-    // FIXME: create and use MountTargetType::RETRACTED
-    if (get_mode() == MAV_MOUNT_MODE_RETRACT) {
-        mnt_target.target_type = MountTargetType::ANGLE;
-        mnt_target.angle_rad.set(Vector3f{0,0,0}, false);
-        send_gimbal_device_retract();
-        return;
-    }
-
-    // send target angles or rates depending on the target type
-    switch (mnt_target.target_type) {
-        case MountTargetType::ANGLE:
-            send_gimbal_device_set_attitude(mnt_target.angle_rad.roll, mnt_target.angle_rad.pitch, mnt_target.angle_rad.yaw, mnt_target.angle_rad.yaw_is_ef);
-            break;
-        case MountTargetType::RATE:
-            send_gimbal_device_set_rate(mnt_target.rate_rads.roll, mnt_target.rate_rads.pitch, mnt_target.rate_rads.yaw, mnt_target.rate_rads.yaw_is_ef);
-            break;
-    }
+    send_target_to_gimbal();
```

Comment:
nitpick, if possible I'd like to have more commenting.  It's pretty obvious but I like it to be **really** obvious :-)